app/pages/common/section/components/fields/bulletpoint-list.njk

```diff
@@ -4,18 +4,17 @@
 {% macro bulletpointList(params) %}
 
   {% if params.isMultiQuestion %}
-    {% set labelClasses = 'nhsuk-u-font-size-19 nhsuk-u-font-weight-bold nhsuk-u-margin-bottom-2' %}
+    {% set labelClasses = 'nhsuk-u-font-size-19 nhsuk-u-font-weight-bold bc-u-inline-block nhsuk-u-margin-bottom-2' %}
   {% else %}
-    {% set labelClasses = 'nhsuk-u-font-size-24 nhsuk-u-font-weight-bold nhsuk-u-margin-bottom-2' %}
+    {% set labelClasses = 'nhsuk-u-font-size-24 nhsuk-u-font-weight-bold bc-u-inline-block nhsuk-u-margin-bottom-2' %}
   {% endif %}
 
   <div data-test-id="question-{{ params.question.id }}">
 
     <div data-test-id="bulletpoint-list-main-advice">
-      {{ label({
-        "text": params.question.mainAdvice,
-        "classes": labelClasses
-      }) }}
+      <span class="{{ labelClasses }}">
```

Comment:
can this be changed to mainAdviceClasses?

Reply:
yeah good point, updated